### PR TITLE
Feature/region documentation

### PR DESCRIPTION
**Pull request by mschellenger**
Add general OCS documentation for regions

### CODE REVIEW

File: Documentation/Management/Account_Tenant.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ uid: AccountTenant
 
 # Tenants
 
-APIs to manage an OCS Customer Tenant Account.
+APIs to manage an OCS Customer Tenant Account. Note that unlinke Namespaces, Tenants are not scoped to a region. Account level information, such as Users and Clients, are stored in all supported regions.
```
Comment (lalanimaulik) [inline Documentation/Management/Account_Tenant.md]: unlike spelling is incorrect

File: Documentation/Management/Account_Namespace_1.md [hunk 2]
```diff
@@ -34,6 +34,18 @@ For HTTP requests and responses, the Namespace object has the following properti
...
+## Region ##
+
+When a Namespace is created, all services are created in the Namespace's region. Consequently, all requests to a Namespace go to the Namespace's region, and all data stored in a Namespace is stored in the Namespace's region. 
+
+| Supported Regions | 
+| --- |
+| WestUS | 
+| WestEurope* |
+
+\* WestEurope Namespaces are currently in preview, and are not yet generally available. If you beleive you have a valid use case for a WestEurope Namespace, please contact OSIsoft.
```
Comment (lalanimaulik) [inline Documentation/Management/Account_Namespace_1.md]: believe spelling is incorrect

File: Documentation/Management/Account_Tenant.md (hunk 1 above)
Comment (mschellenger) [reply Documentation/Management/Account_Tenant.md]: fixed

File: Documentation/Management/Account_Namespace_1.md (hunk 2 above)
Comment (mschellenger) [reply Documentation/Management/Account_Namespace_1.md]: Also fixed. Thanks for catching

File: Documentation/OSIsoft_Cloud_Services.md [hunk 3]
```diff
@@ -33,6 +33,8 @@ Tenants are divided into one or more logical units called Namespaces. Each Names
 other Namespaces, with its own instances of the various OSIsoft Cloud Services and data stores. 
 Namespaces provide isolation of resources. 
 
+Namespaces are provisioned within a region. The Namespace's services and data stores all exist in the Namespace's region. Resources isolated to a Namespace are isolated to the Namespace's region. For more information on regions see the [Namespace](xref:AccountNamespace_1) documentation.
```
Comment (cchengosisoft) [inline Documentation/OSIsoft_Cloud_Services.md]: 'Resources isolated to a Namespace are isolated to the Namespace's region' - too many isolateds.
Maybe you meant to say 'Resources allocated in a Namespace are isolated to the Namespace's region'?

Comment (mschellenger) [reply Documentation/OSIsoft_Cloud_Services.md]: Yeah that sounds much better

File: Documentation/Management/Account_Namespace_1.md [hunk 4]
```diff
@@ -34,6 +34,18 @@ For HTTP requests and responses, the Namespace object has the following properti
 	"State": 0,
 }
 ```
+
+## Region ##
+
+When a Namespace is created, all services are created in the Namespace's region. Consequently, all requests to a Namespace go to the Namespace's region, and all data stored in a Namespace is stored in the Namespace's region.
```
Comment (cchengosisoft) [inline Documentation/Management/Account_Namespace_1.md]: 2nd sentence - 'Consequently, all requests to a Namespace...': I don't think it's clear that this means any resources (streams, topics, etc) at the Namespace level is region scoped - was that the intention?

Comment (mschellenger) [reply Documentation/Management/Account_Namespace_1.md]: How about

"When a Namespace is created, all services are created in the Namespace's region. Resources created in this Namespace (i.e. SDS types, streams, and stream views) will be created in the region of the Namespace, and any data stored in the Namespace will be stored in that region."

File: Documentation/Management/Account_Tenant.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ uid: AccountTenant
 
 # Tenants
 
-APIs to manage an OCS Customer Tenant Account.
+APIs to manage an OCS Customer Tenant Account. Note that unlike Namespaces, Tenants are not scoped to a region. Account level information, such as Users and Clients, are stored in all supported regions.
```
Comment (osimarkrogers) [inline Documentation/Management/Account_Tenant.md]: Remove "Note that".  Just begin with "Unlike Namespaces...."

Comment (mschellenger) [reply Documentation/Management/Account_Tenant.md]: Updated

**Comment by mschellenger** [comment]
@osimarkrogers looks like I don't have permissions to merge, do you have the right permissions or know who does?

**Pull request merged**